main/src/cgeo/geocaching/connector/gc/GCLoggingManager.java

```diff
@@ -67,7 +68,7 @@
     }
 
     @Override
-    public void onLoadFinished(final Loader<String> arg0, final String page) {
+    public void onLoadFinished(@NotNull final Loader<String> arg0, final String page) {
```

Comment:
Shouldn't it be `@NonNull` instead of `@NotNull`?

Reply:
You are right. Is fixed.